Kuzzle.Tests/Kuzzle.Tests.csproj

```diff
@@ -28,4 +28,7 @@
     <Folder Include="API\" />
     <Folder Include="API\Controllers\" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="API\Controllers\BulkController.cs" />

```

Comment:
should be removed: the previous item group already includes the entire API/Controllers folder